src/quo2/components/list_items/saved_contact_address/component_spec.cljs

```diff
@@ -0,0 +1,59 @@
+(ns quo2.components.list-items.saved-contact-address.component-spec
+  (:require [test-helpers.component :as h]
+            [quo2.components.list-items.saved-contact-address.view :as saved-contact-address]
+            [quo2.foundations.colors :as colors]))
+
+(def account
+  {:name                "New House"
+   :address             "0x21a...49e"
+   :emoji               "🍔"
+   :customization-color :purple})
+
+(h/describe "List items: saved contact address"
+  (h/test "default render"
+    (h/render [saved-contact-address/view {:accounts (repeat 1 account)}])
```

Comment:
I'd suggest having one test rendering component without any props. 
Some components tend to crash on this and that makes dev life harder when you try to use them later without knowing correct props.

Reply:
Yeah, this test it supposed to have no props, nice catch! Fixed it